### ISSUE TITLE
[bitnami/thanos] Error creating service account

### ISSUE THREAD

**Issue by barrydobson** [bitnami/charts#13466]
### Name and Version

bitnami/thanos 11.5.9

### What steps will reproduce the bug?

Generating template when creating service account generates error

### Are you using any custom parameters or values?

```yaml
serviceAccount:
  create: true
  name: thanos
```

### What is the expected behavior?

This is a regression. I expect the chart to not error.

### What do you see instead?

```sh
Error: template: thanos/templates/serviceaccount.yaml:5:11: executing "thanos/templates/serviceaccount.yaml" 
at <include "thanos.serviceAccountName" .>: error calling include: template: thanos/templates/_helpers.tpl:332:32: executing "thanos.serviceAccountName" 
at <.context.Values>: nil pointer evaluating interface {}.Values
```

**Comment by hanpeter** [comment]
Also seeing the same issue. This looks relatively new as I didn't run into this with 11.5.7.

**Comment by migruiz4** [comment]
Hi @barrydobson,

Thank you for reporting this issue. I have submitted a fix for this issue in #13502.

Once it is merged, a new version of the `binami/thanos` chart will be released.
